del-geo-core/src/quaternion.rs

```diff
@@ -119,3 +121,15 @@ pub fn from_axisangle(a: &[f32; 3]) -> [f32; 4] {
         (lena * half).cos(),
     ]
 }
+
+pub fn identity() -> [f64; 4] {
```

Comment:
try

```rs
pub const fn identity() -> [f64; 4] {
```

Reply:
cool, I think I will consider use const or inline by the time of doing optimization.